src/main/java/data/schedule/DayList.java

```diff
@@ -1,4 +1,65 @@
 package data.schedule;
 
+import data.plans.Plan;
+import data.plans.PlanList;
+
 public class DayList {
+    private PlanList planList;
+    private String[] scheduleList = new String[7]; //store all plan name for the day
+    private String[] schedulePrintList = new String[7]; //store all plan name for the day to be printed
+    private Day[] dayList = new Day[7]; //store all created day object
```

Comment:
Perhaps is it possible to use just the dayList list instead of having a scheduleList list as the dayList stores Day objects, which stores a Plan object (consisting of the name of the plan)?